### ISSUE TITLE
Keyboard Focus is getting trapped at the "Dark/light" theme buttons:A11y_.NET 9 template_.NET 9 template_Hameburger_Keyboard

### ISSUE THREAD

**Issue by AnKushSingh05** [dotnet/maui#26218]
### GitHub Tags
#A11yTCS; #A11ySev2; #Win32; #WCAG2.1.2; #GH_.NET9template_Win32_Apr2024; #DesktopApp; #.NET Core; #; #A11yMAS; #Keyboard; #No Keyboard Trap:

### Environment Details:
Application Name: .NET 9 template tests that include some 3rd party controls
OS: Windows 11 Enterprise 24H2 (OS build 26100.2314)

### Repro Steps:

1. Install and open ".NET 9 template" app.
2. "Home Screen" will open.
3. Tab till Hamburger menu and press enter,
4. Observe that Keyboard Focus is getting trapped at the "Dark/light" theme buttons

### Actual Result:
Keyboard Focus is getting trapped at the "Dark/light" theme buttons.

### Expected Result:
Keyboard focus should not get trapped in "Dark/light" theme buttons and user should be able to move forward and backward using the tab and shift+ tab keys.

### User Impact:
Keyboard users will find difficulty in going further if keyboard focus gets trapped after the theme buttons.

### Attachment:
https://github.com/user-attachments/assets/e9705ddc-fe2e-4652-83b3-34322081c990

**Comment by jsuarezruiz** [comment]
Blocked by https://github.com/syncfusion/maui-toolkit/issues/67

**Comment by AnKushSingh05** [comment]
Issue is fixed, hence closing the bug.

https://github.com/user-attachments/assets/155a1698-2bad-4d47-84cf-898d4929bcdc
